### PR TITLE
Add Get-CmdletXref

### PR DESCRIPTION
**Pull request by sdwheeler**
# PR Summary

Adds the new Get-CmdletXref command to the MicrosoftDocs module.
- Updated version to 0.0.2
- Included help for cmdlet

## PR Checklist

- [x] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [x] **Summary:** This PR's summary describes the scope and intent of the change.
- [x] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [x] **Style:** This PR adheres to the [style guide][style].

[contrib]: https://github.com/michaeltlombardi/DocumentarianModules/blob/main/CONTRIBUTING.md
[style]: https://github.com/michaeltlombardi/DocumentarianModules/blob/main/CONTRIBUTING.md#Style

### CODE REVIEW

File: Projects/Modules/Documentarian.MicrosoftDocs/CHANGELOG.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ release_links:
 
 ## Unreleased
 
+- Added Get-CmdletXref
```
Comment (michaeltlombardi) [inline Projects/Modules/Documentarian.MicrosoftDocs/CHANGELOG.md:28]: I think we can help folks understand a bit about the change and link them right to the new reference docs:

```suggestion
- Added [`Get-CmdletXref`] to make it simpler to add links to cmdlet reference docs.

[`Get-CmdletXref`]: /modules/microsoftdocs/reference/cmdlets/get-cmdletxref/
```

File: Projects/Modules/Documentarian.MicrosoftDocs/Documentation/reference/cmdlets/Get-CmdletXref.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable,
+-InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose,
+-WarningAction, and -WarningVariable. For more information, see
+[about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
+
+## OUTPUTS
+
+### System.String[]
+
```
Comment (michaeltlombardi) [inline Projects/Modules/Documentarian.MicrosoftDocs/Documentation/reference/cmdlets/Get-CmdletXref.md:106]: Missing docs for the output - this is probably sufficient?

```suggestion
### System.String[]

The cmdlet returns an array of strings, each representing a Markdown link using the Microsoft Learn
platform xref syntax. Each link uses the command name for the link text and the xref string for the
link URI.

```

File: Projects/Modules/Documentarian.MicrosoftDocs/Source/Public/Functions/Get-CmdletXref.ps1 [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    .SYNOPSIS
+    Gets a cross-reference link for a command.
+    #>
+    [CmdletBinding()]
+    [OutputType('System.String[]')]
+    param(
+        # The name of the command to get a cross-reference link for.
+        [Parameter(Mandatory, ValueFromPipeline)]
+        [string[]]$Name
+    )
```
Comment (michaeltlombardi) [inline Projects/Modules/Documentarian.MicrosoftDocs/Source/Public/Functions/Get-CmdletXref.ps1:12]: Nit, but probably useful - Can this be updated to support piping a set of command info objects from `Get-Command`, like:

```powershell
Get-Command Convert*-Json | Get-CmdletXref | Set-Clipboard
```

That might complicate things a bit into two different parameter sets.

**Review by michaeltlombardi — COMMENTED** [review]
Couple notes, suggestions, and a potential enhancement/nit.

File: Projects/Modules/Documentarian.MicrosoftDocs/Source/Public/Functions/Get-CmdletXref.ps1 (hunk 3 above)
Comment (sdwheeler) [reply Projects/Modules/Documentarian.MicrosoftDocs/Source/Public/Functions/Get-CmdletXref.ps1:12]: Good idea. File an issue ;)

**Review by michaeltlombardi — DISMISSED** [review]

**Pull request merged**